l10n_ch_payment_slip/models/payment_slip.py

```diff
@@ -466,12 +466,12 @@ def _draw_address(self, canvas, print_settings, initial_position, font,
         text = canvas.beginText()
         text.setTextOrigin(x, y)
         text.setFont(font.name, font.size)
-        text.textOut(com_partner.name)
+        text.textOut(com_partner.name[:23])
```

Comment:
What about reducing the size of the font instead when the line is longer than 23 characters?

I'm afraid in that case that we would loose valuable informations.
Let say I have a long street name, I could loose the street number.